src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
This **method should return Optional<Manufacturer>**, because there is a possibility to get null if there is no manufacturer with some specific id.
```suggestion
    Optional<Manufacturer> get(Long id);
```